src/main/java/com/sintern/api/DomainController.java

```diff
@@ -0,0 +1,26 @@
+package com.sintern.api;
+
+import com.sintern.domain.Domain;
+import com.sintern.service.OpenInternPositionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
```

Comment:
Since this is a **DomainController**, the base endpoint should be **"/domains"**. The other endpoint will have this **prefixed**. Thus, instead of specifying **value = "/domains"** for the **getDomains()** method, add **@RequestMapping(value = "/domains")** to the class.